content/scout/policy/index.md

```diff
@@ -0,0 +1,172 @@
+---
+title: Get started with policy in Docker Scout
+keywords: scout, supply chain, vulnerabilities, packages, cves, policy
+description: |
+  Policies in Docker Scout lets you define supply chain rules and thresholds
+  for your artifacts, and track how your artifacts perform against those
+  requirements over time
+---
+
+> **Early Access**
+>
+> Docker Scout is in [early access](/release-lifecycle/#early-access-ea). The
+> policy feature is a closed access feature of Docker Scout, and is only
+> available to organizations participating in the early access program for
```

Comment:
Nit: Policy is being released as [Beta](https://docs.docker.com/release-lifecycle/), which is different from Early Access.

Let's change the wording here to something like "...participating in the limited preview program for policies."